Gemfile.lock

```diff
@@ -68,6 +68,7 @@ GEM
     ast (2.4.0)
     bindex (0.7.0)
     builder (3.2.3)
+    byebug (11.0.1)
```

Comment:
I think you need to run `bundle` one more time to update the lockfile so that this isn't included.